charts/nri-prometheus/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 2.7.0
 description: A Helm chart to deploy the New Relic Prometheus OpenMetrics integration
 name: nri-prometheus
-version: 1.8.2
+version: 1.8.3
```

Comment:
This will change some behavior so I'd definitely bump a minor

Reply:
Bumped chart to `1.9.0`